doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +169,113 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
+
+   **Select template option**: In this section, Heptio allows you
+   to click one of two buttons, each runs several commands on AWS
+   that set up a "template" of necessary Kubernetes resources.
+   We recommend ``Option 1``, which will create a new set of resources
+   on AWS to run Kubernetes.
+
+   After clicking the ``Option 1`` button, you'll be taken to an AWS page with a field already
+   chosen under "Choose a template". Simply hit "Next".
+
+   **Enter AWS instance information (page 1)**: On this page you'll need to
+   fill in the following required fields:
+
+   * ``Stack Name`` can be anything you like.
+   * ``Availability Zone`` is related to the location of the AWS
+     resources. Choose an AWS location close to your physical location or
+     any other desired AWS location.
+   * ``Admin Ingress Location`` defines the locations from which you
+     can access this cluster as an administrator. Enter ``0.0.0.0/0``
+     for the most permissive approach.
+   * ``SSH Key`` is a dropdown list of keys attached to your account.
+     The one you created in Step 1 should be listed here. This will allow
+     you to SSH into the machines if you desire.
+   * ``Node Capacity`` defines the number of machines you've got available.
+     This will depend on the ``Instance Type`` that you choose. E.g., if you
+     want each user to have 2GB and you expect 10 users, choose a combination
+     of ``Instance Type`` and ``Node Capacity`` that meets this requirement.
+   * ``Instance Type`` defines what kind of machine you're requesting. See
+     this `list of instance types with Amazon <https://aws.amazon.com/ec2/instance-types/>`_
+     as well as this list of `pricing for each instance type <https://aws.amazon.com/ec2/pricing/on-demand/>`_.
+   * ``Disk Size`` corresponds to the hard disk for each node. If
+     you need users to keep lots of large files that persist over time,
+     this should be larger.
+   * ``Instance Type (Bastion Host)`` corresponds to a "manager"
+     node that coordinates kubernetes. You may leave these as defaults.
+
+   **Enter AWS instance information (page 2)**: On the second page you may leave
+   all of these fields as is or customize as you wish. When done, hit ``Next``. Then
+   confirm and hit ``Next`` once more.
+
+   AWS will now create the computational resources defined in the Heptio
+   template (and according to the options that you chose). This will
+   take a few minutes.
+
+   To see the status of the resources you've requested,
+   see the ``CloudFormation`` page. You should see two stacks being created,
+   each will have the name you've requested. When they're done creating,
+   continue with the guide.
+
+3. Follow Step 3 of the `Heptio guide`_. Install ``kubectl``,
+   and test that your new Kubernetes cluster works properly. In this step
+   you'll need the SSH key file that was generated in Step 1.
+
+4. Create a file, ``storage_cmd.txt`` on your local computer, and enter
```

Comment:
`storageclass.yaml` would be a better name. Also can you really do this on your local computer? Does this not require you do this after sshing?